apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.html

```diff
@@ -1,4 +1,4 @@
-<sky-key-info [layout]="layout">
+<sky-key-info [layout]="layout" data-sky-id="key-info-demo">
   <sky-key-info-value> 575 </sky-key-info-value>
```

Comment:
I know this is splitting hairs - but should we do some sort of logic like we did with "Alert" where maybe we have a `memberCount` variable that then controls the layout. So under 100 is horizontal or something and then we can change that variable and make sure the test updates correctly?

Reply:
updated tests